tooling/bundler/src/bundle/common.rs

```diff
@@ -169,10 +169,12 @@ impl CommandExt for Command {
       let mut lines = stdout_lines_.lock().unwrap();
       loop {
         line.clear();
-        if let Ok(0) = stdout.read_line(&mut line) {
```

Comment:
shouldn't we handle an error here and `continue;` instead?

Reply:
By handling the error, do you mean adding an error message?

E.g.

```rust
match stdout.read_line(&mut line) {
  Ok(0) => break,
  Err(err) => {
    debug!(action = "stdout"; "Err: {}", err);
    continue;
  },
  _ => (),
}
```

Or something else?

Reply:
maybe we should just ignore the error and continue the loop instead of printing that message.

Reply:
Thanks @lucasfernog, I've updated the PR.
It's actually equivalent to my first version of the patch, but much clearer without the `continue` instruction.

I'm using `line.trim_end()` instead of `&line[0..line.len() - 1]` because `line` does not end with `\n` if the stream ends without line break, as I wrote above.